src/modifier/modifier-type.ts

```diff
@@ -1767,16 +1777,10 @@ export function getPlayerModifierTypeOptions(
   const options: ModifierTypeOption[] = [];
   const retryCount = Math.min(count * 5, 50);
   if (!customModifierSettings) {
-    new Array(count).fill(0).map((_, i) => {
-      options.push(
-        getModifierTypeOptionWithRetry(
-          options,
-          retryCount,
-          party,
-          modifierTiers && modifierTiers.length > i ? modifierTiers[i] : undefined,
-        ),
-      );
-    });
+    for (let i = 0; i < count; i++) {
+      const tier = modifierTiers && modifierTiers.length > i ? modifierTiers[i] : undefined;
```

Comment:
```suggestion
      const tier = modifierTiers?.length > i ? modifierTiers[i] : undefined;
```

although doing 
```suggestion
      const tier = modifierTiers?[i]; 
```
would also do the same I think?

Reply:
Can't do the first one:
<img width="955" height="24" alt="image" src="https://github.com/user-attachments/assets/8d53226c-0579-4100-a7cd-c9de5146f191" />
The second should work I believe, based on a quick test.

Reply:
I right, it would need tiers.length ?? 0 to work